### PR TITLE
Socket Connections

### PR DESCRIPTION
**Pull request by MitchPetersen**
Completed command transfer between webservice and transaction server, added redis distributable, began testing of redis.

Commands will need to be rejiggered now that I know how redis actually works. Won't take much work.

### CODE REVIEW

File: Scripts/Transactions.py [hunk 1]
```diff
@@ -223,9 +229,85 @@ def setSellTrigger(userID, stockSymbol, amount):
...
+        else:
+            response = "Unknown command"
+            print(data)
+            print("received unknown command")
+
+        # Format response
+        if response == 1:
+            response = {"status": 200}
+        else:
+            response = {"status": 500, "message": response}
```
Comment (MitchellGRead) [inline Scripts/Transactions.py]: could we change the key to reason here instead of message

File: Scripts/Connections.py [hunk 2]
```diff
@@ -8,11 +8,22 @@
...
 dbName, dbUser, dbPassword = "TBD"
 
 
+# Creates socket for Sending/Receiving from WebService
+def connectWeb():
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.bind((localHost, transPort))
+    s.listen()
+    conn, addr = s.accept()
+    print("Connected to WebService @ " + addr[0])
```
Comment (MitchellGRead) [inline Scripts/Connections.py]: maybe just add the port as well so we can be double sure the connection is as we expect

**Review by MitchellGRead — CHANGES_REQUESTED** [review]
Do we need to have all those binary files that seem to be in the PR or can we gitignore them?

File: Scripts/Transactions.py (hunk 1 above)
Comment (MitchPetersen) [reply Scripts/Transactions.py]: Yep.

File: Scripts/Connections.py (hunk 2 above)
Comment (MitchPetersen) [reply Scripts/Connections.py]: I can add most of them to the gitignore, the only one really need is redis-server. I don't know if it relies on anything.

**Review by MitchPetersen — COMMENTED** [review]
Changes made.

**Review by MitchellGRead — APPROVED** [review]

**Pull request merged**